release/terraform/main.tf

```diff
@@ -0,0 +1,55 @@
+terraform {
+  required_providers {
+    google = {
+      source  = "hashicorp/google"
+      version = "4.51.0"
+    }
+  }
+}
+
+provider "google" {
+  project = "bqutil"
```

Comment:
Dynamically populate project depending on which environment is running (test vs prod)